helm/test-values.yaml

```diff
@@ -0,0 +1,249 @@
+      password: ERzCT4pwBDxfCKRGmfrMa8KQ8sXf8GKy
+
+##TODO:
+#1. UPDATE WRONGSECRETS TO DEFINITION WHERE K8S IS USED FOR AWS!
+#2. UPDATE VIRTUALDESKTOP TO DEFINITION WHERE IT CN BE USED
+#3. ADD VAULT INSTANCE PER TEAM!
+
+wrongsecrets:
+  # -- Specifies how many Wrongsecrets instances should start at max. Set to -1 to remove the max Wrongsecrets instance cap
+  maxInstances: 500
+  # -- Wrongsecrets Image to use
+  image: jeroenwillemsen/wrongsecrets
+  tag: 1.6.4-no-vault
+  # -- Change the key when hosting a CTF event. This key gets used to generate the challenge flags. See: https://github.com/OWASP/wrongsecrets#ctf
+  ctfKey: "zLp@.-6fMW6L-7R3b!9uR_K!NfkkTr"
+  # -- Specify a custom Juice Shop config.yaml. See the JuiceShop Config Docs for more detail: https://pwning.owasp-juice.shop/part1/customization.html#yaml-configuration-file
+  # @default -- See values.yaml for full details
+  config: |
+    K8S_ENV: aws
+  #    application:
+  #      logo: https://raw.githubusercontent.com/iteratec/multi-juicer/main/images/multijuicer-icon-only-padding.png
+  #      favicon: https://raw.githubusercontent.com/iteratec/multi-juicer/main/wrongsecrets-balancer/ui/public/favicon.ico
+  #      showVersionNumber: false
+  #      showGitHubLinks: false
+  #    challenges:
+  #      showHints: true
+  #    hackingInstructor:
+  #      isEnabled: true
+  #    ctf:
+  #      showFlagsInNotifications: false
+  # -- Specify a custom NODE_ENV for JuiceShop. If value is changed to something other than 'wrongsecrets-ctf-party' it's not possible to set a custom config via `juiceShop.config`.
+  nodeEnv: "wrongsecrets-ctf-party"
+  # -- Optional resources definitions to set for each JuiceShop instance
+  resources:
+    requests:
+      cpu: 256Mi
+      memory: 300Mi
+  #  limits:
+  #    cpu: 100m
+  #    memory: 200Mi
+  # -- Optional securityContext definitions to set for each JuiceShop instance
+  securityContext:
+    allowPrivilegeEscalation: false
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
+    capabilities:
+      drop:
+        - ALL
+    seccompProfile:
+      type: RuntimeDefault
+  # -- Optional environment variables to set for each JuiceShop instance (see: https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/)
+  env:
+    - name: K8S_ENV
+      value: k8s
+    - name: SPECIAL_K8S_SECRET
+      valueFrom:
+        configMapKeyRef:
+          name: secrets-file
+          key: funny.entry
+    - name: SPECIAL_SPECIAL_K8S_SECRET
+      valueFrom:
+        secretKeyRef:
+          name: funnystuff
+          key: funnier
+  # env:
+  #   - name: FOO
+  #     valueFrom:
+  #       secretKeyRef:
+  #         key: FOO
+  #         name: secret-resource
+  # -- Optional mount environment variables from configMaps or secrets (see: https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#configure-all-key-value-pairs-in-a-secret-as-container-environment-variables)
```

Comment:
As discussed on slack: let's make sure this all is configurable and readable from a config map I think? What do you think @bendehaan ?

Reply:
It's read by the configMap from the `values.yaml` file.